src/Extractor/RequestRpcExtractor.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimPod\JsonRpc\Extractor;
+
+use Psr\Http\Message\RequestInterface;
+
+final class RequestRpcExtractor extends RpcExtractor
```

Comment:
nedává my smysl mít u všech metod "Rpc" - co jiného by to bylo? :D je to redundant ... platí pro všechny tyhle extractor classes

Reply:
tohle sis asi pořádně nepřečetl, psal jsem metody, ne třídy

Reply:
aha, mi i tridy :D prejmenuju